### PR TITLE
Adds Inventory labels

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This PRs adds functionality so a user can add labels to an inventory in the same way that they can add labels to a job template.  

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - API
 - UI

##### ADDITIONAL INFORMATION
inventoryLabels

### CODE REVIEW

**Review by keithjgrant — APPROVED** [review]

File: awx/main/access.py
Comment (AlanCoding) [inline awx/main/access.py:861]: In the access class `can_attach` and `can_unattach` should get called when managing labels... but it's likely the case that `BaseAccess` logic is sufficient for this.

**Review by nixocio — CHANGES_REQUESTED** [review]
Hey @AlexSCorey, overall great job.

labels

There is an extra time that no labels are displayed when editing the inventory.

**Review by nixocio — APPROVED** [review]
Approve UI changes. I triggered the e2e tests.

File: awx/api/views/inventory.py
Comment (jbradberry) [inline awx/api/views/inventory.py:21]: The `Label` model is re-imported at the top level of awx.main.models, so it can be put up in the big list above instead of as a separate line here.

**Review by jbradberry — APPROVED** [review]

**Comment by jbradberry** [comment]
@AlexSCorey oh, I see that you have to update the unit tests.

**Comment by AlanCoding** [comment]
Is this blocked on anything? As far as I can tell things look ready to go.

**Comment by AlexSCorey** [comment]
@AlanCoding I am just waiting on testing. Just moved the issue to Needs test.

File: awx/main/tests/unit/models/test_label.py
Comment (rebeccahhh) [inline awx/main/tests/unit/models/test_label.py:24]: good change

**Review by rebeccahhh — APPROVED** [review]

**Comment by InnocentK** [comment]
@tiagodread and @akus062381 After this PR is merged we will need to add test coverage for this new feature

**Comment by InnocentK** [comment]
Passes UI E2E testing: https://dashboard.cypress.io/projects/ui2mq7/runs/11658/overview (failed test is unrelated).
Will still need to verify API E2E testing before merging

**Comment by InnocentK** [comment]
Passes API testing

**Pull request merged**
